images/metering-ansible-operator/build/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM quay.io/operator-framework/ansible-operator:v0.6.0
```

Comment:
Move this file to the root of the repo and name it Dockerfile.metering-ansible-operator, and update the contents to COPY the roles/watches.yaml from `images/metering-ansible-operator/roles` and `images/metering-ansible-operator/watches.yaml`.

Reply:
You have two copies of the Dockerfile. This needs to be removed.

Reply:
This appears to be unresolved since last review